felix/ipsets/ipsets.go

```diff
@@ -681,6 +692,9 @@ func (s *IPSets) resyncIPSet(ipSetName string) error {
 		return scanner.Err()
 	})
 	if err != nil {
+		s.setNameToProgrammedMetadata.Dataplane().Set(ipSetName, dataplaneMetadata{
+			ListFailed: true,
+		})
```

Comment:
It's possible to use an empty meta struct here. But thought to add a field to be more explicit.